tests/test_rdpg_corr.py

```diff
@@ -0,0 +1,49 @@
+import unittest
+from graspy.simulations.simulations import sample_edges, rdpg, p_from_latent
+from graspy.simulations.rdpg_corr import rdpg_corr
+import numpy as np
+import pytest
+import warnings
+
+
+class Test_RDPG_Corr(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.r = 0.3
+        cls.Y = None
+        cls.X = np.random.dirichlet([20, 20], size=300)
+
+    def test_dimensions(self):
+        A, B = rdpg_corr(
+            self.X, self.Y, self.r, rescale=False, directed=False, loops=False
+        )
+        self.assertTrue(A.shape, (300, 300))
+        self.assertTrue(B.shape, (300, 300))
+
+    def test_inputs(self):
+        x1 = np.array([[1, 1], [1, 1]])
+        x2 = np.array([[1, 1]])
+        x3 = np.zeros((2, 2, 2))
+        with self.assertRaises(TypeError):
+            p_from_latent("hi")  # wrong type
+        with self.assertRaises(ValueError):
+            p_from_latent(x1, x2)  # dimension mismatch
+        with self.assertRaises(ValueError):
+            p_from_latent(x3)  # wrong num dimensions
+        with self.assertRaises(TypeError):
+            sample_edges("XD")  # wrong type
+        with self.assertRaises(ValueError):
+            sample_edges(x3)  # wrong num dimensions
+        with self.assertRaises(ValueError):
+            sample_edges(x2)  # wrong shape for P
+
+    def test_rdpg_corr(self):
+        g1, g2 = rdpg_corr(
+            self.X, self.Y, self.r, rescale=False, directed=False, loops=False
+        )
+
+        # check rho
```

Comment:
@SHAAAAN add a test to check P makes sense as well.

Reply:
@SHAAAAN can you point me to where this is addressed?

Reply:
@SHAAAAN what i meant was, a test demonstrating that the output values of P after sampling graphs are close to those those that we expect based on the input matrices. Does that make sense? I think you are talking about input checks, which is good, but not what I am asking about.

Reply:
as in, you assert that the correlation is correct, but you don't assert that P is correct

Reply:
I see what you mean now. Can I just copy the code from the test in rdpg for p?